kpi/security/ir.model.access.csv

```diff
@@ -4,8 +4,8 @@
 "access_kpi_category_user","kpi.category.user","model_kpi_category","base.group_user",1,0,0,0
 "access_kpi_threshold_user","kpi.threshold.user","model_kpi_threshold","base.group_user",1,0,0,0
 "access_kpi_threshold_range_user","kpi.threshold.range.user","model_kpi_threshold_range","base.group_user",1,0,0,0
-"access_kpi_manager","kpi.manager","model_kpi","base.group_user",1,1,1,1
-"access_kpi_category_manager","kpi.category.manager","model_kpi_category","base.group_user",1,1,1,1
-"access_kpi_threshold_manager","kpi.threshold.manager","model_kpi_threshold","base.group_user",1,1,1,1
-"access_kpi_threshold_range_manager","kpi.threshold.range.manager","model_kpi_threshold_range","base.group_user",1,1,1,1
+"access_kpi_manager","kpi.manager","model_kpi","kpi.group_kpi_manager",1,1,1,1
+"access_kpi_category_manager","kpi.category.manager","model_kpi_category","kpi.group_kpi_manager",1,1,1,1
+"access_kpi_threshold_manager","kpi.threshold.manager","model_kpi_threshold","kpi.group_kpi_manager",1,1,1,1
```

Comment:
I think there's a missing line for access_kpi_history_manager:
"access_kpi_history_manager","kpi.history.user","model_kpi_history","kpi.group_kpi_manager",1,1,1,1

Reply:
> Sure!, but maybe the user should be able to create also the history if he can refresh the data.
Just a question, all users should be able to access the KPIs? Or a group should be created?